src/tracker.jl

```diff
@@ -11,6 +11,10 @@ function second_lowest(::TrackerBackend)
     return throw(ArgumentError("Tracker backend does not support nested differentiation."))
 end
 
+primal_value(x::Tracker.TrackedReal) = Tracker.data(x)
+primal_value(x::Tracker.TrackedArray) = Tracker.data(x)
+primal_value(x::AbstractArray{<:Tracker.TrackedReal}) = Tracker.data.(x)
```

Comment:
This line does not appear to be hit by the tests. In lieu of trial-and-error to try to get this branch hit, a simple test for this function specifically would work.

Reply:
test added for this method, not through an AD example though because I am not sure when this method will be needed but it might sense to define